lib/LocalizationProvider.js

```diff
@@ -35,6 +35,8 @@ module.exports = LocalizationProvider;
 var util = require('util');
 
 var LOCALE_COOKIE_KEY = 'locale',
+	LOCALE_COOKIE_MAX_AGE = 3155692600, // 100 years
```

Comment:
Please name with prefix `DEFAULT_`. This approach is used everywhere in Catbery.